docs/MEMORY-SETTINGS.md

```diff
@@ -220,6 +220,36 @@ Changing these parameters is recommended as a last resort, for example to debug
 an OOM kill due to a bloated `zedbox` process. Before changing the values,
 please read the [documentation](https://tip.golang.org/doc/gc-guide) carefully.
 
+## Forced execution of the Golang runtime garbage collector settings
+
+Setting a soft limit for Golang garbage collector may not be enough:
+the whole system may experience memory pressure due to other
+applications in the same of other memory cgroups. In such cases, GC
+can be forced from the memory pressure event handler. The following
+parameters are available for configuring this algorithm in order to
+minimize frequent GC calls:
+
+1. `gogc.forced.period.seconds` sets period of forced GC loop in seconds,
+    meaning that GC is called explicitly o more than once every 10 seconds.
+    Default value is 10 seconds.
+
+2. `gogc.forced.gained.mem.MiB` sets absolute amount of gained (allocated)
+   memory since last reclaim, meaning that GC will be called again only
+   if desired amount is gained. Default value is 50 MiB.
+
```

Comment:
See above about "gained"

Reply:
This is gained, the memory which was allocated by the app since last GC call, so we assume, that if something increased, then probably we do need to call GC once again, because system still experiences memory pressure and this guy (an application) already gained (allocated) something.